sycl/test/check_device_code/const_ref_image_accessor.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clangxx -fsycl -fsycl-device-only %s -o /dev/null
```

Comment:
BTW, you 
```suggestion
// RUN: %clangxx -fsycl -fsycl-device-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning %s
// expected-no-diagnostics
```
This one should be even better.